src/components/CardPlay/index.js

```diff
@@ -28,13 +29,14 @@ const CardPlay = (props) => {
   }
   return (
     <div className="card-container">
-      <img className="image" src="https://via.placeholder.com/70"></img>
+      <ImageAudio />
       <div className="audio-title">
-        <p className="title">{title}</p>
-        <span className="icon-play" onClick={toggleAudio}>{isPlaying ? (<PauseCircleTwoTone style={{ fontSize: "25px" }} />) : (<PlayCircleTwoTone style={{ fontSize: "25px" }} />)}</span>
+        <p>{title}</p>
+        <span className="icon-play">{isPlaying ? (<PauseCircleTwoTone style={{ fontSize: "25px" }} />) : (<PlayCircleTwoTone style={{ fontSize: "25px" }} />)}</span>
         <div className="audio">
           <div className="player">
             <div className="logo" onClick={toggleAudio}>
+
             </div>
```

Comment:
Si este elemento no va a contener nada, puede escribirse así:
```suggestion
            <div className="logo" onClick={toggleAudio} />
```
Ahora, si va a contener algo, pero aún no está hecho ese algo, te recomendaría poner un comentario en el código para que se entienda que ahí va algo.

Reply:
Tomo la corrección del cierre de etiqueta, la cuestión es que ese div pertenece a la rama base la cual estaba trabajando Nico, por eso lo dejo tal cual.

Reply:
No problemo, veo igual que la rama base ya se unió a development, así que podrías meterlo como fix en un PR posterior.